Fail to build when ENABLE_EXT_C=0 ENABLE_JIT=1

When the configuration ENABLE_EXT_C is set to 0 and ENABLE_JIT is set to 1, it fails to build:
```
In file included from src/jit.c:1314:
src/rv32_jit.c:609:50: error: no member named 'shamt' in 'struct rv_insn'
emit_alu32_imm8(state, 0xc1, 5, temp_reg[0], ir->shamt);
                                             ~~  ^
src/jit.c:1312:9: note: expanded from macro 'GEN'
        code;                                                                 \
        ^~~~
In file included from src/jit.c:1314:
src/rv32_jit.c:614:50: error: no member named 'shamt' in 'struct rv_insn'
emit_alu32_imm8(state, 0xc1, 7, temp_reg[0], ir->shamt);
                                             ~~  ^
src/jit.c:1312:9: note: expanded from macro 'GEN'
        code;                                                                 \
        ^~~~
2 errors generated.
make: *** [build/jit.o] Error 1
```

The way to reproduce:
```
make clean && make ENABLE_EXT_C=0 ENABLE_JIT=1
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -34,6 +34,9 @@ jobs:
     - name: JIT test
       run: |
             make ENABLE_JIT=1 clean check
+            make ENABLE_EXT_A=0 ENABLE_JIT=1 clean check
+            make ENABLE_EXT_C=0 ENABLE_JIT=1 clean check
+            make ENABLE_EXT_F=0 ENABLE_JIT=1 clean check
 
   host-arm64:
     runs-on: ubuntu-22.04
@@ -59,6 +62,9 @@ jobs:
           make
           make check
           make ENABLE_JIT=1 clean check
+          make ENABLE_EXT_A=0 ENABLE_JIT=1 clean check
+          make ENABLE_EXT_C=0 ENABLE_JIT=1 clean check
+          make ENABLE_EXT_F=0 ENABLE_JIT=1 clean check
 
   coding-style:
     runs-on: ubuntu-22.04

--- a/tools/gen-jit-template.py
+++ b/tools/gen-jit-template.py
@@ -110,7 +110,7 @@ def parse_argv(EXT_LIST, SKIP_LIST):
     for ext in EXT_LIST:
         SKIP_LIST += INSN[ext]
 
-# parse_argv(EXT_LIST, SKIP_LIST)
+parse_argv(EXT_LIST, SKIP_LIST)
 # prepare PROLOGUE
 output = ""
 f = open('src/rv32_template.c', 'r')
@@ -125,7 +125,7 @@ codegen_funcs = re.findall(r'GEN\([\s|\S]+?}\)', lines)
 op = []
 impl = []
 for i in range(len(emulate_funcs)):
-    op.append(emulate_funcs[i][5:emulate_funcs[i].find(',')])
+    op.append(emulate_funcs[i][5:emulate_funcs[i].find(',')].strip())
     impl.append(codegen_funcs[i])
 
 f.close()

